src/plugins/intel_gpu/src/runtime/debug_configuration.cpp

```diff
@@ -121,6 +121,7 @@ static void print_help_messages() {
     message_list.emplace_back("OV_GPU_ForceImplType", "Force implementation type of a target primitive or layer. [primitive or layout_name]:[impl_type]"
                               "For primitives, fc:onednn, fc:ocl, do:cpu, do:ocl, reduce:ocl and reduce:onednn are supported");
     message_list.emplace_back("OV_GPU_MaxKernelsPerBatch", "Maximum number of kernels in a batch during compiling kernels");
+    message_list.emplace_back("OV_GPU_TestDeviceIndex", "Set test device by index");
```

Comment:
I suggest removing "test" word from the option name

Reply:
* IMO, having "test" is OK because this will be used only for unit test / func test purpose. BTW, we need to clarify it from the help message..
* I think this should be exposed even when DEBUG_CAPS is disabled. So this needs to be a separate env variable. Any idea @vladimir-paramuzov ?

Reply:
@isanghao 
1. we may use it for tests only, but the option will work in any app. We don't prohibit usage like `OV_GPU_TestDeviceIndex=2 ./benchmark_app -d GPU.1 ...`
2. Why do we need that?

Reply:
1. OK
2. I expect CI test on the system where DG2 HW is installed, and the system might have iGPU as well. Then, we need this option.

Reply:
Removed 'Test' prefix